src/main/resources/api/service_openapi.yaml

```diff
@@ -155,6 +155,28 @@ paths:
         200:
           $ref: '#/components/responses/QueryResponse'
 
+  /api/v1/unique-values/{version}:
```

Comment:
You need to update the version in `info` at the top when a new API is added. We haven't been good about this in the past, but we should be better about it in the future.

Reply:
incremented to 1.1.1